configure.ac

```diff
@@ -56,6 +56,7 @@ AC_CHECK_HEADERS([sys/select.h sys/time.h pty.h util.h termios.h])
 AC_CHECK_LIB([network],[socket])
 AC_CHECK_LIB([util],[forkpty])
 AC_CHECK_LIB([wolfssl],[wolfCrypt_Init],,[AC_MSG_ERROR([libwolfssl is required for ${PACKAGE}. It can be obtained from https://www.wolfssl.com/download.html/ .])])
+AC_CHECK_LIB([wolftpm],[wolfTPM2_Init],,[AC_MSG_ERROR([libwolftpm is required for ${PACKAGE}. It can be obtained from https://www.wolfssl.com/download.html/ .])])
```

Comment:
This is being required all the time. It needs to be in the `AC_ARG_ENABLE([tpm],` section.

Reply:
This check is moved under the conditional check and now the build fails with ... when "--enable-tpm" is set.
```
Undefined symbols for architecture x86_64:
  "_wolfTPM2_SignHashScheme", referenced from:
      _SendKexDhReply in libwolfssh_la-internal.o
      _SendUserAuthRequest in libwolfssh_la-internal.o
```

Any idea why autoconf behaves like this?